cli/auth_nkey_command.go

```diff
@@ -58,6 +60,19 @@ func configureAuthNkeyCommand(auth commandHost) {
 	nkVerify.Arg("file", "File containing the data to check").Required().ExistingFileVar(&c.dataFile)
 	nkVerify.Arg("signature", "File containing the signature").Required().ExistingFileVar(&c.signFile)
 	nkVerify.Arg("key", "The key to use for verification").Required().ExistingFileVar(&c.keyFile)
+
+	nkSeal := nk.Command("seal", "Encrypts file").Alias("encrypt").Alias("enc").Action(c.sealAction)
+	nkSeal.Arg("file", "File to sign").Required().ExistingFileVar(&c.dataFile)
+	nkSeal.Arg("key", "NKey to sign with").Required().ExistingFileVar(&c.keyFile)
+	nkSeal.Arg("receipent", "Public XKey of receipient").Required().StringVar(&c.counterpartKey)
+	nkSeal.Flag("output", "Write the encrypted data to a file").StringVar(&c.outFile)
+	nkSeal.Flag("b64", "Write base64 encoded data [Default]").Default("true").BoolVar(&c.b64out)
```

Comment:
looks like the unseal doesnt handle base 64 encoded files so we need a similar option on the unseal side?